components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/listeners/GatewayJMSMessageListener.java

```diff
@@ -27,10 +27,7 @@
 import org.apache.commons.logging.LogFactory;
 import org.wso2.carbon.apimgt.api.APIManagementException;
 import org.wso2.carbon.apimgt.api.model.APIStatus;
-import org.wso2.carbon.apimgt.gateway.APILoggerManager;
-import org.wso2.carbon.apimgt.gateway.EndpointCertificateDeployer;
-import org.wso2.carbon.apimgt.gateway.GoogleAnalyticsConfigDeployer;
-import org.wso2.carbon.apimgt.gateway.InMemoryAPIDeployer;
+import org.wso2.carbon.apimgt.gateway.*;
```

Comment:
Shall we revert this change to remove the wild card import? Let's change the applicable other places as well.